matchms/similarity/CosineGreedy.py

```diff
@@ -1,10 +1,12 @@
 from typing import Tuple
-import numpy
+from matchms.similarity.spectrum_similarity_functions import collect_peak_pairs
+from matchms.similarity.spectrum_similarity_functions import get_peaks_array
+from matchms.similarity.spectrum_similarity_functions import score_best_matches
```

Comment:
These import are a bit long, use relative imports
```suggestion
from .spectrum_similarity_functions import collect_peak_pairs
from .spectrum_similarity_functions import get_peaks_array
from .spectrum_similarity_functions import score_best_matches
```

Reply:
Thanks that looks better indeed.